lldb/source/Plugins/ExpressionParser/Swift/SwiftUserExpression.cpp

```diff
@@ -94,177 +94,181 @@ static CompilerType GetConcreteType(ExecutionContext &exe_ctx,
   return type;
 }
 
+/// Determine whether we have a Swift language symbol context. This handles
+/// some special cases, such as when the expression language is unknown, or
+/// when we have to guess from a mangled name.
+static bool isSwiftLanguageSymbolContext(const SwiftUserExpression &expr,
+                                         const SymbolContext &sym_ctx) {
+  if (sym_ctx.comp_unit && (expr.Language() == lldb::eLanguageTypeUnknown ||
+                            expr.Language() == lldb::eLanguageTypeSwift)) {
+    if (sym_ctx.comp_unit->GetLanguage() == lldb::eLanguageTypeSwift ||
+        sym_ctx.comp_unit->GetLanguage() == lldb::eLanguageTypePLI)
```

Comment:
We should really remove the PL1 line. This dates back to a brief period in 2013 before DW_LANG_Swift was defined and before DW_LANG_APPLE_Swift could be defined due to upstream LLVM contstraints.